src/lib/services/search-actions/AasSearcher.ts

```diff
@@ -100,7 +100,7 @@ export class AasSearcher {
             return this.createAasResult(defaultResult);
         }
 
-        const potentiallyMultipleAas = await this.getAasFromAllRepositories(aasIdEncoded);
+        const potentiallyMultipleAas = await this.shellsFromAllRepositories(aasIdEncoded);
         if (potentiallyMultipleAas) {
             if (potentiallyMultipleAas.length === 1) {
```

Comment:
Sorry, I wasn't quite clear. I think the method name was fine, I just didn't like the variable name of `potentiallyMultipleAas`

Reply:
potentiallyMultipleAas seems to be a good name for me because valid results are 0, 1, ore more Aas

Reply:
I'm with Fleanegan on this one!